client/component/src/ConnectedPlot.tsx

```diff
@@ -79,81 +79,213 @@ const defaultAxesParameters = {
+ * @interface {object} BatonRequestMessage
+ * @member {string} uuid - The universally unique identifier.
+ */
 interface BatonRequestMessage {
+  /** The universally unique identifier */
   uuid: string;
 }
 
+/**
+ * A clear plots message.
+ * @interface {object} ClearPlotsMessage
+ * @member {string} plot_id - The plot ID.
+ */
 interface ClearPlotsMessage {
+  /** The plot ID */
   plot_id: string;
 }
 
+/**
+ * A data message.
+ * @interface {object} DataMessage
+ * @member {AxesParameters} axes_parameters - The axes parameters.
+ */
 interface DataMessage {
+  /** The axes parameters */
   axes_parameters: AxesParameters;
 }
 
+/**
+ * A multiline data message.
+ * @interface {object} MultiLineDataMessage
+ * @extends {DataMessage}
+ * @member {LineData[]} ml_data - The multiline data.
+ */
 interface MultiLineDataMessage extends DataMessage {
+  /** The multiline data */
   ml_data: LineData[];
 }
 
+/**
+ * An append line data message.
+ * @interface {object} AppendLineDataMessage
+ * @extends {DataMessage}
+ * @member {LineData[]} al_data - The line data to append.
+ */
 interface AppendLineDataMessage extends DataMessage {
+  /** The line data to append */
   al_data: LineData[];
 }
 
+/**
+ * An image data message.
+ * @interface {object} ImageDataMessage
+ * @extends {DataMessage}
+ * @member {ImageData} im_data - The image data.
+ */
 interface ImageDataMessage extends DataMessage {
+  /** The image data */
   im_data: ImageData;
 }
 
+/**
+ * A scatter data message.
+ * @interface {object} ScatterDataMessage
+ * @extends {DataMessage}
+ * @member {ScatterData} sc_data - The scatter data.
+ */
 interface ScatterDataMessage extends DataMessage {
+  /** The scatter data */
   sc_data: ScatterData;
 }
 
+/**
+ * A surface data message.
+ * @interface {object} SurfaceDataMessage
+ * @extends {DataMessage}
+ * @member {SurfaceData} su_data - The surface data.
+ */
 interface SurfaceDataMessage extends DataMessage {
+  /** The surface data */
   su_data: SurfaceData;
 }
 
+/**
+ * A table data message.
+ * @interface {object} TableDataMessage
+ * @extends {DataMessage}
+ * @member {TableData} ta_data - The table data.
+ */
 interface TableDataMessage extends DataMessage {
+  /** The table data */
   ta_data: TableData;
 }
 
+/**
+ * The props for the `ConnectedPlot` component.
+ * @interface {object} ConnectedPlotProps
+ * @member {string} plot_id - The plot ID.
+ * @member {string} hostname - The hostname.
+ * @member {string} port - The port.
+ * @member {string} uuid - The uuid.
+ */
 interface ConnectedPlotProps {
+  /** The plot ID */
   plot_id: string;
+  /** The hostname */
   hostname: string;
+  /** The port */
   port: string;
+  /** The universally unique identifier */
   uuid: string;
 }
 
+/**
+ *
+ * Renders a connected plot.
+ * Needs a python plot server running. This can be run from the top level with
+ * "uvicorn --factory davidia.main:create_app"
+ * and tested with
+ * "PYTHONPATH=. python davidia/demos/simple.py"
```

Comment:
Probably not the best place to add this to show up in the storybook.

Reply:
This partly works but there's no "plot_1" so the demo stops early! Perhaps that `ValueError` should just be a warning...